src/rust_lexer/Cargo.lock

```diff
@@ -0,0 +1,6 @@
+# This file is automatically @generated by Cargo.
+# It is not intended for manual editing.
```

Comment:
Hmm, we should have a `.gitignore` rule for these - either way, this shouldn't exist.
Even if you run `cargo` in a subdirectory, it should see the workspace top-level `Cargo.lock`.